thirdparty/s3.go

```diff
@@ -0,0 +1,37 @@
+package thirdparty
+
+import (
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+//PresignExpireTime sets the amount of time the link is live before expiring
+const PresignExpireTime = 24 * time.Hour
+
+//PreSign returns a presigned url that expires in 24 hours
+func PreSign(bucket string, filekey string, awsKey string, awsSecret string) (string, error) {
```

Comment:
can this be a struct with key names, 4 strings in a row is a bit hard to read and easy to get wrong for callers.